docs/node-mixin/rules/rules.libsonnet

```diff
@@ -0,0 +1,116 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'node-exporter.rules',
+        rules: [
+          {
+            // This rule gives the number of CPUs per node.
+            record: 'instance:node_num_cpu:sum',
+            expr: |||
+              count without (cpu) (
+                sum without (mode) (
```

Comment:
count may be a tad cheaper, and we don't care about the sum

Reply:
Changed.